static/loginerror.html.template

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>{{ app.title }}</title>
+    <link rel="stylesheet" type="text/css" href="/static/login.css" />
+  </head>
+  <body>
+    <div class="loginBox">
+      <div class="loginHeader">
+        <div class="logos">
+            {%- for logo in app.logos %}
```

Comment:
@stefanv I explored this, but I think it would require changes to baselayer. We'd need to change how we initialize jinja2 template objects to specify a FileLoader with access to the base login page template file in order to extend it in the error page. Might be more trouble than it's worth? https://stackoverflow.com/questions/6195805/jinja2-render-template-inheritance